python_cross_site_scripting/dekoza_django-minishop/admin.py

```
@@ -0,0 +1,133 @@
+                        app_dict[app_label]['models'].append(model_dict)
+                    else:
+                        app_dict[app_label] = {
+                            'name': app_label.title(),
+                            'app_url': app_label + '/',
+                            'has_module_perms': has_module_perms,
+                            'models': [model_dict],
+                        }
+
+        # Sort the apps alphabetically.
+        app_list = app_dict.values()
+        app_list.sort(lambda x, y: cmp(x['name'], y['name']))
```

Ruff (0.13.3)

Undefined name `cmp`

(F821)